crates/pallet-domains/src/weights.rs

```diff
@@ -57,47 +57,53 @@ pub struct SubstrateWeight<T>(PhantomData<T>);
-	/// Storage: `Domains::LatestConfirmedDomainBlock` (r:1 w:1)
-	/// Proof: `Domains::LatestConfirmedDomainBlock` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	/// Storage: `Domains::NewAddedHeadReceipt` (r:1 w:1)
+	/// Proof: `Domains::NewAddedHeadReceipt` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	/// Storage: `Domains::LatestConfirmedDomainExecutionReceipt` (r:1 w:1)
+	/// Proof: `Domains::LatestConfirmedDomainExecutionReceipt` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	/// Storage: `Domains::BlockTree` (r:2 w:2)
 	/// Proof: `Domains::BlockTree` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	/// Storage: `Domains::BlockTreeNodes` (r:1 w:2)
 	/// Proof: `Domains::BlockTreeNodes` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	/// Storage: `Domains::LatestSubmittedER` (r:1 w:0)
 	/// Proof: `Domains::LatestSubmittedER` (`max_values`: None, `max_size`: None, mode: `Measured`)
-	/// Storage: `Domains::ExecutionInbox` (r:2 w:1)
+	/// Storage: `Domains::ExecutionInbox` (r:3 w:1)
 	/// Proof: `Domains::ExecutionInbox` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	/// Storage: `Transporter::DomainBalances` (r:1 w:1)
 	/// Proof: `Transporter::DomainBalances` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	/// Storage: `Domains::DomainRegistry` (r:1 w:0)
+	/// Proof: `Domains::DomainRegistry` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	/// Storage: `Domains::DomainRuntimeUpgradeRecords` (r:1 w:0)
+	/// Proof: `Domains::DomainRuntimeUpgradeRecords` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	/// Storage: `Domains::SuccessfulBundles` (r:1 w:1)
 	/// Proof: `Domains::SuccessfulBundles` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	/// Storage: `Domains::HeadDomainNumber` (r:1 w:1)
 	/// Proof: `Domains::HeadDomainNumber` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	/// Storage: `Domains::OperatorBundleSlot` (r:1 w:1)
+	/// Proof: `Domains::OperatorBundleSlot` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	/// Storage: `Domains::InboxedBundleAuthor` (r:0 w:1)
 	/// Proof: `Domains::InboxedBundleAuthor` (`max_values`: None, `max_size`: None, mode: `Measured`)
-	/// Storage: `Domains::HeadReceiptExtended` (r:0 w:1)
-	/// Proof: `Domains::HeadReceiptExtended` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	/// Storage: `Domains::ConsensusBlockHash` (r:0 w:1)
 	/// Proof: `Domains::ConsensusBlockHash` (`max_values`: None, `max_size`: None, mode: `Measured`)
 	fn submit_bundle() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `1629`
-		//  Estimated: `7569`
-		// Minimum execution time: 85_000_000 picoseconds.
-		Weight::from_parts(89_000_000, 7569)
-			.saturating_add(T::DbWeight::get().reads(11_u64))
-			.saturating_add(T::DbWeight::get().writes(13_u64))
+		//  Measured:  `1751`
+		//  Estimated: `10166`
+		// Minimum execution time: 150_394_000 picoseconds.
```

Comment:
Any idea why this has almost doubled in time?

Reply:
My guess is:
- CPU changed, the previous benchmarks were running on mac chip
- The benchmark hasn't run for a while and there are maybe some changes in the code that bring more weight.

Reply:
>The benchmark hasn't run for a while and there are maybe some changes in the code that bring more weight.

This does seem related, I run the benchmark on my machine and it is `Weight::from_parts(119_000_000, 10166)` now.